entity/machine.go

```diff
@@ -83,6 +83,7 @@ type Machine struct {
 	DisableIPv4                  bool                `json:"disable_ipv4,omitempty"`
 	Netboot                      bool                `json:"netboot,omitempty"`
 	Locked                       bool                `json:"locked,omitempty"`
+	EnableHwSync                 bool                `json:"enable_hw_sync,omitempty"`
```

Comment:
MAAS doesn't expose such field. 
https://github.com/maas/maas/blob/54ef20d60e9dccdf7fea80c218e142eaf771ec37/src/maasserver/api/machines.py#L120

However there are other exposed properties that are related to hardware sync functionality:

- last_sync
- sync_interval
- next_sync

Maybe they can be used to deduct if hardware sync is enabled for the machine?

Reply:
I will remove this property as part of this proposal and will add it in a separate PR.